docs/api/wrapper/setChecked.md

```diff
@@ -1,9 +1,19 @@
-## setChecked(value)
+## setChecked(checked)
 
-Sets the value of a radio or checkbox `<input>`.
+Sets the value to an input element of type checkbox or an input element of type radio.
```

Comment:
I think we should change this to mention the v-model:

`Sets checked value for input element of type checkbox or radio and updates `v-model` bound data.`